packages/office-addin-manifest/src/manifestHandler/getManifestHandler.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ManifestHandler } from "./manifestHandler";
+import { ManifestHandlerJson } from "./manifestHandlerJson";
+import { ManifestHandlerXml } from "./manifestHandlerXml";
+
+function isJsonFile(path: string) {
+  return path.endsWith(".json");
+}
+
+export function getManifestHandler(manifestPath: string): ManifestHandler {
```

Comment:
Maybe this should be part of ManifestInfo?  Is it used anywhere else besides that?

Reply:
The main reason is to abstract the `ManifestInfo` file from importing the specific manifest types.
By having this in a different file, the `ManifestInfo` doesn't have to know which handler it is dealing with (not importing them)